perfkitbenchmarker/benchmarks/hadoop_terasort_benchmark.py

```diff
@@ -36,21 +36,23 @@
 FLAGS = flags.FLAGS
 
 BENCHMARK_INFO = {'name': 'hadoop_terasort',
-                  'description': 'Runs Terasort',
+                  'description': 'Runs Terasort. Control the number of VMs '
+                  'with --num_vms.',
```

Comment:
It is, actually - the results of `GetInfo` are used to specify both the number of VMs run by default, and whether they require a scratch volume. Relevant section of `pkb.py --help`:
```
Benchmarks (default requirements):
...
        hadoop_terasort: Runs Terasort. Control the number of VMs with --num_vms. (9 VMs with scratch volume)
...
```